.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# General summary
+
+<!-- Please describe the PR content -->
+
```

Comment:
How about we add a section where the developer can add screenshots or gifs ?

From a personal view, it is really useful to see how the change has impacted the UI with a screenshot or a gifs, maybe showing the before or after.